src/style.css

```diff
@@ -1 +1,91 @@
-/* Styles go here */
+* {
+  padding: 0;
+  margin: 0;
+  list-style: none;
+  box-sizing: border-box;
+}
```

Comment:
do not reset everything here , it wont inherit from here 
instead only use `box-sizing: border-box;` here 
you can have style using `body` tag and reset `margin` there
`list-style: none` should be done for the class where you have your list item